readme.md

```diff
@@ -188,8 +188,32 @@ This example shows all options with their default options.
 }
 ```
 
+
+## PatternPack Workflow
+
+### Pattern Library Development
+When developing new patterns for a pattern library, PatternPack provides the `default` and `build` tasks to assist with the process.  The `default` task is primarily used for interactive development.  It hosts a simple webserver for reviewing changes, and will automatically compile CSS and markdown into patterns as changes are made.
+
+The `build` task does not run the webserver or monitor for changes.  It is best used for manual updates and inspection of the pattern library.  It is also useful to call as part of a customized build process.
+
+### Pattern Library Release
+In order to release a new version of a pattern library you create with PatternPack, the following sequence of commands should be executed.
+
+```
+grunt build
+grunt release
```

Comment:
Sherif ran into some mis-documentation about these. Technically `grunt build` and `grunt release` would have to be configured in your parent project.

In your parent project, if you configure patternpack as such:
```
module.exports = function(grunt) {
  grunt.initConfig({
    // The default patternpack task to generate patterns
    patternpack: {
      options: {
        ...
      },
      run: {},
      build: {
        options: {
          task: 'build'
        }
      },
      release: {
        options: {
          task: 'release'
        }
      }
    }
  });
}
```

Then technically your tasks would be `grunt patternpack:build` and `grunt patternpack:release`. You can integrate those into your project's build and release tasks.